easybuild/easyconfigs/r/rocminfo/rocminfo-5.6.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'rocminfo'
+version = '5.6.0'
+
+homepage = 'https://github.com/RadeonOpenCompute/rocminfo'
+description = """ROCm Application for Reporting System Info"""
+docurls = ['https://rocmdocs.amd.com']
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+github_account = 'RadeonOpenCompute'
+source_urls = [GITHUB_SOURCE]
+sources = ["rocm-%(version)s.tar.gz"]
+checksums = ['87d98a736e4f7510d1475d35717842068d826096a0af7c15a395bcf9d36d7fa0']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+]
+dependencies = [
+    ('Clang-AOMP', '%(version)s'),
+    ('Python', '3.10.4'),  # Needed for 'rocm_agent_enumerator' which is a Python script
+]
+
+configopts = "-DCMAKE_EXE_LINKER_FLAGS='-Wl,--disable-new-dtags'"
+configopts += " -DROCRTST_BLD_TYPE=Release"
+configopts += " -DROCM_DIR=%(installdir)s"
+
+sanity_check_paths = {
+    'files': ['bin/rocminfo', 'bin/rocm_agent_enumerator'],
+    'dirs': [],
+}
+sanity_check_commands = [
+    'rocminfo --help',
```

Comment:
This fails for me:
```
$ rocminfo --help
ROCk module is loaded
Unable to open /dev/kfd read-write: Permission denied
branfosj is not member of "video" group, the default DRM access group. Users must be a member of the "video" group or another DRM access group in order for ROCm applications to run successfully.
```

Not sure if this means this package is unsuitable for an easyconfig or if we sould skip this sanity check.

Reply:
Just as for CUDA the AMDGPU device needs proper permissions.

We have this:
```
cat /etc/udev/rules.d/71-amdgpu.rules 
# Fix AMD GPU device permissions
#
# This file is managed by puppet, local changes will be overwritten.
#
SUBSYSTEM=="drm", KERNEL=="renderD*", GROUP="render", MODE="0666"
SUBSYSTEM=="kfd", GROUP="render", MODE="0666"
```

Reply:
Without it, you would need to explicitly add all users that want to use it to the "video" group